spark_parser/spark.py

```diff
@@ -494,11 +500,16 @@ def makeSet(self, token, sets, i):
 
             if parent == i:
                 continue
-
+            if self.debug['reduce']: #STRANIKS_SCAN 01/03/2016
+                indent2 = len(self.states[state].complete)
             for rule in self.states[state].complete:
                 lhs, rhs = rule
                 if self.debug['reduce']:
-                    print("%s ::= %s" % (lhs, ' '.join(rhs)))
+                    text1 = "%d %d%s%s%s ::= %s" % (indent1, indent2, '   '*indent1, ' '*indent2, lhs, ' '.join(rhs))
```

Comment:
It would help me if you fleshed out an example of this kind of debug output versus what is there now so I can see how this is more useful. Thanks.